docs/GraFx-Studio/concepts/design-run/index.md

```diff
@@ -0,0 +1,63 @@
+# Design Mode and Run Mode
+
+GraFx Studio provides two modes for creating and testing Smart Templates: **Design Mode** and **Run Mode**. 
+
+These modes allow you to focus separately on designing the smart template and testing business logic.
+
+In **Design Mode**, you can freely arrange and style elements without triggering actions or interactions. This keeps the workspace clean, ideal for setting up and adjusting layouts, styles, and initial logic.
+
+Switching to **Run Mode** simulates the end-user experience (My Projects), showing how the template’s logic behaves live. In this mode, actions and visibility settings behave as they will in a real environment, letting you test how users will experience the template.
+
+## Design Mode
+
+**Design Mode** is optimized for layout and style tasks, with logic and actions inactive.
+
+![screenshot](dr1.png)
+
+![screenshot-full](dr1_1.png)
+
+### Key Features in Design Mode:
+- **Static Layout**: Actions and logic are inactive, letting you set up without interference.
+- **Focused Styling**: Only layout and style elements are active, making design adjustments easier.
```

Comment:
I would go a bit more in-depth:
You can create text, image, barcode, and shape frames
You can create a Stylekit with font families, paragraph and character styles, and colors